src/video_utils.py

```diff
@@ -204,6 +261,16 @@ def download_video(url):
                 break  # Exit the loop once the file is found
     except subprocess.CalledProcessError as e:
         error("Error downloading video: %s", e)
+        if "[Instagram]" in str(e) and "No video formats found!" in str(e):
+            debug("yt-dlp failed for Instagram URL, trying gallery-dl")
+        try:
+            result_path = download_instagram_media(url, temp_dir)
```

Comment:
Suggestion: naming `download_instagram_images()`, since the video is also a media.

Reply:
Done